README.md

```diff
@@ -1,10 +1,166 @@
-# rimrep-examples
+# Automated data access from the Reef 2050 Integrated Monitoring and Reporting Program Data Management System (RIMReP DMS)
 
-This repository contains example notebooks in `R` and `Python` showing how to access datasets available in the Reef 2050 Integrated Monitoring and Reporting Program Data Management System (RIMReP DMS).  
+This repository contains example notebooks in `R` and `Python` showing how to access datasets available in the Reef 2050 Integrated Monitoring and Reporting Program Data Management System (RIMReP DMS).  These notebooks include suggested workflows on how to query datasets to create summary tables, figures, and maps.  
   
-Example notebooks also include suggested workflows on how to query datasets to create summary tables, figures, and maps.  
+## What is RIMReP DMS?
+RIMReP DMS is an Open Geospatial Consortium (OGC) API service and analysis-ready, cloud-optimised (ARCO) repository for data and metadata relevant to the management of the Great Barrier Reef. RIMReP DMS offers services to allow the discovery of the data and the interaction with external RIMReP systems.  
+  
+In simple terms, RIMReP DMS is a data portal that aims to be a "one-stop-shop" for all data related to the Great Barrier Reef World Heritage Area, which can be easily accessed by the Great Barrier Reef Marine Park Authority (GBRMPA) to support evidence-based management strategies. All datasets have a standard format regardless of their origin, which not only facilitates access to data, but also their analysis as it removes the need to quality control individual datasets.  
 
-## Setting up your machine
+Datasets that have an open licence are publically available in the RIMReP DMS, while datasets that have a restricted licence are only available to users that have been granted access to them by the data provider.  
+  
+## Searching for datasets in RIMReP DMS
+There are two main ways to browse datasets available in the RIMReP DMS:  
+  
+1. Via the SpatioTemporal Asset Catalogs (STAC). STAC provides a common language to describe a range of geospatial information, so that data can be indexed and easily discovered. Our STAC catalogue is available at [https://stac.staging.reefdata.io/](https://stac.staging.reefdata.io/).  
+2. Via Pygeoapi, which provides API access to geospatial data that is compliant with OGC API standards. Our Pygeoapi service is available at [https://pygeoapi.staging.reefdata.io/](https://pygeoapi.staging.reefdata.io/).  

```

Comment:
I think we should present STAC as the only way to search for datasets. We expect people to get the link to specific datasets on pygeoapi after looking them up on STAC, presenting them together like this can confuse people as to which one they are supposed to use.

Reply:
Agree. We must encourage the user to use STAC-browser